texthero/representation.py

```diff
@@ -442,9 +324,13 @@ def pca(s, n_components=2, random_state=None) -> pd.Series:
     In general, *pca* should be called after the text has already been
     represented to a matrix form.
 
+    PCA cannot directly handle sparse input, so when calling pca on a
+    DocumentTermDF, the input has to be expanded which can lead to
+    memory problems with big datasets.
+
     Parameters
     ----------
-    s : Pandas Series
+    s : Pandas Series (VectorSeries) or MultiIndex Sparse DataFrame (DocumentTermDF)
```

Comment:
MultiIndex Sparse DataFrame (DocumentTermDF): DataFrame is enough (change that also in the other functions below)